post-install.yml

```diff
@@ -0,0 +1,146 @@
+- hosts: "{{ hosts_prefix }}-tsuru-api*[0]"
+  sudo: no
+  tasks:
+    - name: Tsuru target-list
+      shell: >
+        tsuru target-list
+      register: tsuru_targets
+    - name: Tsuru add target.
+      shell: >
+        tsuru target-add {{ deploy_env }}_{{ domain_name }} http://127.0.0.1:8080
+      when: "not '{{ deploy_env }}_{{ domain_name }}' in tsuru_targets.stdout"
+    - name: Tsuru set target.
+      shell: >
+        tsuru target-set {{ deploy_env }}_{{ domain_name }}
+    - name: Tsuru login
```

Comment:
The best reason I could think of for the other one using sudo is the apt package installation - I'm not sure if it's actually required for apt installs - It's certainly not needed for the rest of the steps being carried out on that host.